testapi.pm

```diff
@@ -604,8 +510,62 @@ sub check_var_array {
     return grep { $_ eq $val } @$vars_r;
 }
 
-## helpers
+=head1 script execution helpers
+
+=head2 wait_serial
+
+  wait_serial($regex or ARRAYREF of $regexes [[, $timeout_sec], $expect_not_found]);
```

Comment:
I prefer to name variables here same as the actual name, i.e. either say "$timeout" or rename variable within code to "$timeout_sec"